backend/models/feedbackModel.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const feedbackSchema = mongoose.Schema(
+  {
+    body: {
```

Comment:
Also a potential satisfaction rating? (The Figma diagram indicates a satisfaction rating)
![image](https://user-images.githubusercontent.com/24628243/161669259-98050a32-5292-4055-9039-71b3949b5af3.png)

Reply:
Fixed: Changes are reflected in recent commits